back/src/domain/offer/useCases/SendBeneficiariesAssesmentsEmails.ts

```diff
@@ -0,0 +1,111 @@
+import { addDays } from "date-fns";
+import { z } from "zod";
+import {
+  castError,
+  ConventionDto,
+  ConventionId,
+  immersionFacileNoReplyEmailSender,
+} from "shared";
+import { CreateNewEvent } from "../../core/eventBus/EventBus";
+import { TimeGateway } from "../../core/ports/TimeGateway";
+import { UnitOfWork, UnitOfWorkPerformer } from "../../core/ports/UnitOfWork";
+import { TransactionalUseCase } from "../../core/UseCase";
+import { SaveNotificationAndRelatedEvent } from "../../generic/notifications/entities/Notification";
+
+type SendBeneficiaryAssessmentEmailsOutput = {
+  errors?: Record<ConventionId, Error>;
+  numberOfImmersionEndingTomorrow: number;
+};
+
+export class SendBeneficiariesAssessmentsEmails extends TransactionalUseCase<
```

Comment:
Qu'est-ce que vous pensez qu'on se trouve un autre nom pour éviter de confondre avec l'assessment justement ? 
`LegacyPdfAssesment` peut-être ?

```suggestion
export class SendBeneficiariesLegacyPdfAssessmentsEmails extends TransactionalUseCase<
```